src/lidar/read_rp_lidar.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <RPLidar/rplidar.h>
+
+#include "../navtypes.h"
+#include "../world_interface/data.h"
+
+namespace lidar {
+
+bool initializeLidar();
+
+void readLidarLoop();
+
+typedef struct Polar2D {
+	double r, theta;
+	double asCartesianX() {
+		return r * cos(theta);
+	}
+	double asCartesianY() {
+		return r * sin(theta);
+	}
+} Polar2D;
```

Comment:
TODO: We might want to clean this up in the (near) future - this class is duplicated between RPLidar and Hokuyo Lidar. Maybe we could include casting operators in the class to ease conversion between Polar2D and PointXY (although for the latter we should be using `navtypes::point_t`).

Reply:
Sounds good, there is some redundancy that should be cleaned up.